JumpscaleCore/data/schema/JSXObject.py

```diff
@@ -219,6 +219,14 @@ def _check(self):
         self._ddict
         return True
 
+    def __setattr__(self, name, value):
+        name_condition = not name.startswith("_") and name not in ["acl_id", "nid"]
+
+        if name_condition and name not in self._properties_names_get():
+            raise AttributeError(f"Object has no attribute {name}")
+
+        j.baseclasses.object.__setattr__(self, name, value)
+
```

Comment:
should not be done here, is part of baseclass Attr.py
thats where the test is done for protection

Reply:
well, actually the jsxobject doesn't inherit from Attr.py at any point. it inherits only from JSBase, that's why this logic wasnt applied to it. But it's applied just fine to the bcdbconfig object because they inherit from Attr.py. I added an inheritance from Attr.py to the jsxobject class, but it's currently barfing in my face with an infinite loop, so I am trying to figure out what's the problem.